pkg/archive/diff.go

```diff
@@ -240,11 +240,13 @@ func applyLayerHandler(dest string, layer io.Reader, options *TarOptions, decomp
 	dest = filepath.Clean(dest)
 
 	// We need to be able to set any perms
-	oldmask, err := system.Umask(0)
-	if err != nil {
-		return 0, err
+	if runtime.GOOS != "windows" {
+		oldmask, err := system.Umask(0)
+		if err != nil {
+			return 0, err
+		}
+		defer system.Umask(oldmask) // ignore err, ErrNotSupportedPlatform
```

Comment:
The "ErrNotSupportedPlatform" was from Windows (see `pkg/system/umask_windows.go`), so this comment no longer makes sense.

Reply:
Will remove